### PR TITLE
Move windows build from appveyor to azure

### PR DESCRIPTION
**Pull request by SeptimiuVana**

### CODE REVIEW

File: cmake/aditof-setup.iss.cmakein [hunk 1]
```diff
@@ -15,7 +15,7 @@ AppSupportURL="http://www.analog.com"
 AppUpdatesURL="http://www.analog.com"
 AppCopyright="Copyright 2019 Analog Devices, Inc."
 CreateAppDir=yes
-LicenseFile="C:\projects\aditof-sdk\LICENSE"
+LicenseFile="D:\a\1\s\LICENSE"
```
Comment (dNechita) [inline cmake/aditof-setup.iss.cmakein:18]: You can get the path to the repo by using cmake. This way to get rid of harcoded paths. Without harcoded paths, one can easily generate it's own installed on his/her pc.

File: cmake/aditof-setup.iss.cmakein [hunk 2]
```diff
@@ -58,11 +53,11 @@ Name: "ukrainian"; MessagesFile: "compiler:Languages\Ukrainian.isl"
 Name: "desktopicon"; Description: "{cm:CreateDesktopIcon}"; GroupDescription: "{cm:AdditionalIcons}"
 
 [Files]
-Source: "C:\projects\aditof-sdk\build\sdk\Release\aditof.dll"; DestDir: "{app}"
-Source: "C:\projects\aditof-sdk\build\examples\aditof-demo\Release\aditof-demo.exe"; DestDir: "{app}";
+Source: "D:\a\1\s\build_Release\sdk\Release\aditof.dll"; DestDir: "{app}"
```
Comment (dNechita) [inline cmake/aditof-setup.iss.cmakein:56]: You can get the path to the build folder using cmake.

File: cmake/aditof-setup.iss.cmakein (hunk 1 above)
Comment (SeptimiuVana) [reply cmake/aditof-setup.iss.cmakein]: Fixed.

File: cmake/aditof-setup.iss.cmakein (hunk 2 above)
Comment (SeptimiuVana) [reply cmake/aditof-setup.iss.cmakein]: Fixed.

**Review by dNechita — APPROVED** [review]

**Pull request merged**